lib/ansible/modules/windows/win_service.py

```diff
@@ -183,30 +196,30 @@
     name: service name
     state: restarted
     username: NT AUTHORITY\NetworkService
-    password: ""
+    password: ''
 
 - name: Set the log on user to Local Service
   win_service:
     name: service name
     state: restarted
     username: NT AUTHORITY\LocalService
-    password: ""
+    password: ''
 
 - name: Set dependencies to ones only in the list
   win_service:
     name: service name
-    dependencies: ['service1', 'service2']
+    dependencies: [ service1, service2 ]
```

Comment:
should this be a yaml list instead, same with the other changes below?

Reply:
This is also a valid YAML list, both are possible.
I am not in favor to mandate using the expanded (multi-line) format.
In fact I think there's merit here in switching between both syntaxes in subsequent examples.

Reply:
I'll put this on the list of things to discuss in the Documentation WG.
To me, the documentation examples serve a few purposes:
- Demonstrate possible ways to use the module first and foremost
- Show good practices (like always using `name:`)
- Learn possible constructs/syntax in YAML
- Do not give wrong impressions (by using a minimalistic YAML syntax, e.g. indentation, quoting)

cc @acozine